### PR TITLE
Add about me section and favicon

### PR DESCRIPTION
**Pull request by anamatasanch**
+About me -> aboutme
+Favicon-> favicon

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -38,9 +40,40 @@
...
+                  <p style="font-size:1vw;">Bachelor of Science in Computer Science</p>
+                </div>
+              </div>
+            </div>
         </div>
         <div class="contact-container">
+          <div class='name-container-content'>
+            <ul>
+              <li><img src="https://img.icons8.com/small/16/000000/secured-letter.png"/>
+                <a>Almatasanch@miners.utep.edu</a>
```
Comment (joanapascoal) [inline portfolio/src/main/webapp/index.html]: Usually it should be in lower case.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -38,9 +40,40 @@
...
+              </li>
+              <li><img src="https://img.icons8.com/small/16/000000/secured-letter.png"/>
+                <a>Ana.mata7892@gmail.com</a>
```
Comment (joanapascoal) [inline portfolio/src/main/webapp/index.html]: Same here.

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -1,9 +1,11 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta charset="UTF-8">
     <title>Ana Mata's Portfolio</title>
     <link rel="stylesheet" href="css/style.css">
+    <link rel="icon" href="Images/icon.png">
```
Comment (joanapascoal) [inline portfolio/src/main/webapp/index.html]: Images folder should be lower case.

**Review by joanapascoal — APPROVED** [review]
Good job! 

Just a few minor comments.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (anamatasanch) [reply portfolio/src/main/webapp/index.html]: Fixed!

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (anamatasanch) [reply portfolio/src/main/webapp/index.html]: Fixed!

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (anamatasanch) [reply portfolio/src/main/webapp/index.html]: Fixed!

**Pull request merged**